taskimages/activestandby/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the binary
```

Comment:
it may also be worth looking at the structure of the builder stage in this dockerfile - and make it match the others - just purely a neatness thing?

Reply:
Updated the structure to match the other Dockerfiles